leaf-ui/js/onFunctions.js

```diff
@@ -1343,3 +1348,9 @@ function checkForMultipleNB(node) {
 	return num >= 1;
 }
 
+function loadAnalysisConfig(){
```

Comment:
Is this func exclusively for loading the first config when you switch to analysis mode, or do you imagine that it could be used for loading additional configs from a json? Just clarifying

Reply:
Also good practice to docstring all functions

Reply:
For now its exclusively for loading the first config but my hope is in the future once we do loading and saving it can be used to check for additional configs and load all of them. Docstring added!